rllib/tuned_examples/bc/benchmark_atari_pong_bc.py

```diff
@@ -0,0 +1,257 @@
+"""
+schema={
```

Comment:
Is this needed here? If this is an explanation of the schema, maybe move this down into the config section (where we explain the schema translation)?